packages/core/components/DraggableComponent/index.tsx

```diff
@@ -109,12 +118,20 @@ export const DraggableComponent = ({
     [loadedOverrides]
   );
 
+  const permissions =
+    selectedItem &&
+    getPermissions({
+      selectedItem,
+      globalPermissions: globalPermissions || {},
+      config,
+    });
+
   return (
     <El
       key={id}
       draggableId={id}
       index={index}
-      isDragDisabled={isDragDisabled}
+      isDragDisabled={!permissions?.drag || isDragDisabled}
```

Comment:
@chrisvxd This isn't working if another component is selected and I drag the Hero in the custom UI. How can I get the dragging component's state, not the selected one?